### PR TITLE
Fix tests for different doctrine versions

### PR DESCRIPTION
**Pull request by JohanLopes**

### CODE REVIEW

**Comment by stollr** [comment]
Hm, I am not sure if the solution with the $aliasSeparator is the best. It makes the string less readable. I'd prefer something like this:

``` php
if (version_compare(\Doctrine\ORM\Version::VERSION, '2.5', '<')) {
    $this->assertEquals('SELECT a0_.id AS id0, a0_.title AS title1 FROM Article a0_ ORDER BY a0_.title ASC LIMIT 10 OFFSET 0', $executed[1]);
    $this->assertEquals('SELECT a0_.id AS id0, a0_.title AS title1 FROM Article a0_ ORDER BY a0_.title DESC LIMIT 10 OFFSET 0', $executed[3]);
} else {
    $this->assertEquals('SELECT a0_.id AS id_0, a0_.title AS title_1 FROM Article a0_ ORDER BY a0_.title ASC LIMIT 10 OFFSET 0', $executed[1]);
    $this->assertEquals('SELECT a0_.id AS id_0, a0_.title AS title_1 FROM Article a0_ ORDER BY a0_.title DESC LIMIT 10 OFFSET 0', $executed[3]);
}
```

And please add a short comment and then rebase.

**Comment by JohanLopes** [comment]
I have put changes asking by @naitsirch (Ping @leonex-cstoller)

File: tests/Test/Pager/Subscriber/Sortable/Doctrine/ORM/QueryTest.php [hunk 1]
```diff
@@ -95,8 +95,17 @@ function shouldSortSimpleDoctrineQuery()
 
         $this->assertEquals(4, $this->queryAnalyzer->getNumExecutedQueries());
         $executed = $this->queryAnalyzer->getExecutedQueries();
-        $this->assertEquals('SELECT a0_.id AS id_0, a0_.title AS title_1 FROM Article a0_ ORDER BY a0_.title ASC LIMIT 10 OFFSET 0', $executed[1]);
-        $this->assertEquals('SELECT a0_.id AS id_0, a0_.title AS title_1 FROM Article a0_ ORDER BY a0_.title DESC LIMIT 10 OFFSET 0', $executed[3]);
+
+        $aliasSeparator = version_compare(\Doctrine\ORM\Version::VERSION, '2.5', '<') ? "" : "_";
```
Comment (leonex-cs1) [inline tests/Test/Pager/Subscriber/Sortable/Doctrine/ORM/QueryTest.php]: Did you forget to remove this `$aliasSeparator = ...`?

File: tests/Test/Pager/Subscriber/Sortable/Doctrine/ORM/QueryTest.php [hunk 2]
```diff
@@ -164,7 +179,15 @@ function shouldWorkWithInitialPaginatorEventDispatcher()
 
         $this->assertEquals(2, $this->queryAnalyzer->getNumExecutedQueries());
         $executed = $this->queryAnalyzer->getExecutedQueries();
-        $this->assertEquals('SELECT a0_.id AS id_0, a0_.title AS title_1 FROM Article a0_ ORDER BY a0_.title ASC LIMIT 10 OFFSET 0', $executed[1]);
+
+        $aliasSeparator = version_compare(\Doctrine\ORM\Version::VERSION, '2.5', '<') ? "" : "_";
```
Comment (leonex-cs1) [inline tests/Test/Pager/Subscriber/Sortable/Doctrine/ORM/QueryTest.php]: Did you forget to remove this `$aliasSeparator = ...`?

**Comment by JohanLopes** [comment]
Exactly, i forget to remove these lines... ;(
I fixed that and make a rebase

**Comment by leonex-cs1** [comment]
Looks good. Thank you very much!

**Pull request merged**